docs/root/intro/comparison.rst

```diff
@@ -27,7 +27,7 @@ proxy:
   run a mix of nginx and haproxy. A single proxy solution at every hop is substantially simpler from
   an operations perspective.
 
-`haproxy <http://www.haproxy.org/>`_
```

Comment:
several title underlines are too short, and that is why the docs runs is erroring

Reply:
Right. Thank you for your review.